src/components/block-tunes/block-tune-move-down.ts

```diff
@@ -109,6 +109,9 @@ export default class MoveDownTune implements BlockTune {
     /** Change blocks positions */
     this.api.blocks.move(currentBlockIndex + 1);
 
+    this.api.toolbar.open();
+    this.api.toolbar.toggleBlockSettings(true);
```

Comment:
Looks not really convenient, maybe we can at least use just one method?

Reply:
done